{"name": "Process a phone order with Save and then Pay", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' == True\n +  where 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' = order('Phone Order')\n +    where order = <logic.takeout_logic.Calculations object at 0x0000020A29546600>.order\n +      where <logic.takeout_logic.Calculations object at 0x0000020A29546600> = <test_phoneorder_card.TestPhoneOrderCard object at 0x0000020A2C6C3590>.phone", "trace": "self = <test_phoneorder_card.TestPhoneOrderCard object at 0x0000020A2C6C3590>\n\n    @allure.story(\"Phone Order Save and Pay\")\n    @allure.title(\"Process a phone order with Save and then Pay\")\n    def test_phoneorder_save_pay(self):\n    \n>       assert self.phone.order(\"Phone Order\") == True\nE       AssertionError: assert 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' == True\nE        +  where 'ERROR IS Message: \\nStacktrace:\\nNoSuchElementError: An element could not be located on the page using the given sear...ata\\\\Roaming\\\\npm\\\\node_modules\\\\appium\\\\node_modules\\\\@appium\\\\base-driver\\\\lib\\\\basedriver\\\\commands\\\\find.ts:75:12)' = order('Phone Order')\nE        +    where order = <logic.takeout_logic.Calculations object at 0x0000020A29546600>.order\nE        +      where <logic.takeout_logic.Calculations object at 0x0000020A29546600> = <test_phoneorder_card.TestPhoneOrderCard object at 0x0000020A2C6C3590>.phone\n\nsuits\\test_orderflows\\test_card\\test_phoneorder_card.py:29: AssertionError"}, "start": 1741780465844, "stop": 1741780475993, "uuid": "a595cc29-9548-4bc7-b00e-a3326005c1ac", "historyId": "9979c322a3eae085832f011ab6e7b51b", "testCaseId": "9979c322a3eae085832f011ab6e7b51b", "fullName": "suits.test_orderflows.test_card.test_phoneorder_card.TestPhoneOrderCard#test_phoneorder_save_pay", "labels": [{"name": "feature", "value": "Sanity Sheet"}, {"name": "story", "value": "Phone Order Save and Pay"}, {"name": "parentSuite", "value": "suits.test_orderflows.test_card"}, {"name": "suite", "value": "test_phoneorder_card"}, {"name": "subSuite", "value": "TestPhoneOrderCard"}, {"name": "host", "value": "LAPTOP-KKMKONCC"}, {"name": "thread", "value": "15076-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "suits.test_orderflows.test_card.test_phoneorder_card"}]}